model/response.py

```diff
@@ -44,6 +43,10 @@ def map_reactions_list(map_path):
 class Response(object):
     def __init__(self, model, message, wild_type_model_id=None):
         self.model = model
+        self.old_objective = self.model.objective
+        if constants.OBJECTIVE in message:
+            if model.reactions.has_id(message[constants.OBJECTIVE]):
```

Comment:
IIUC, this safeguard ensures that if an unknown objective is requested, we ignore it and optimize for the default. But does that really make sense? Is it valid for the frontend to request an unrecognized reaction? If not, maybe it is better to simply let this raise an exception.